ebusd-2.1.x/en/vaillant/08.hmu.csv

```diff
@@ -12,27 +12,31 @@ r,,RunDataCompressorOutletTemp,,,,,A208,,,tempv,,,,,,,,,,,,,,,,,,,,,
 r,,RunDataEEVOutletTemp,,,,,AC08,,,tempv,,,,,,,,,,,,,,,,,,,,,
```

Comment:
why are some in this area named rundata and other runstats?

Reply:
If I remember, I think that is just trying to use naming that is consistent with (inconsistent) naming from multiple other branches. I don't have a strong preference myself, only that there is some sort of overall naming convention.